.github/workflows/cicd.yaml

```diff
@@ -16,14 +16,25 @@ jobs:
       - run: ./bin/make ci
         env:
           TERM: vt100
+
+  deploy:
+    runs-on: ubuntu-latest
+    environment: stage
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
       - run: ./bin/make deploy
         env:
           FIREBASE_SERVICE_ACCOUNT: ${{ secrets.FIREBASE_SERVICE_ACCOUNT_EVY_LANG }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - run: ./bin/make e2e
+        env:
+          BASEURL: ${{env.BASEURL_APEX}}
```

Comment:
nit: inconsistent spacing with the other github expressions (secrets) above - spaces bracket the expression

Reply:
yup, fixed.